### DISCUSSION TITLE
Hosting server defaults to 0.0.0.0. instead of 127.0.0.1

### DISCUSSION THREAD

**Discussion by Mitchell-Crumbley** [nss-evening-cohort-14/evening-client-side#8] [Help Me!] [answered]
Name: Mitchell Crumbley

When I run the command hs -o in my terminal it opens a server but it is from http://0.0.0.0:8080 instead of 127.0.0.1:8080. I'm unable to connect to 0.0.0.0 and view if the changes I'm making in our Star Wars files are sticking or not. 

List any error messages that appear
"The operation was rejected by your operating system.
npm ERR! It is likely you do not have the permissions to access this file as the current user"

What have you googled? 
1- https://github.com/http-party/http-server/pull/132/commits/678ac62991bdde703862352d559fdfbcb776abad
2 - https://github.com/http-party/http-server/pull/129
3 - https://stackoverflow.com/questions/43889497/chrome-cannot-connect-to-http-localhost8080-but-http-127-0-0-18080-works

What have you tried? Tell us everything you have tried.
I tried uninstalling and reinstalled node. 
I have running a number of hs commands including http-server -a 127.0.0.1, http-server -v

**Comment by TrinityChristiana** [comment]
Please use the template defined here: https://github.com/nss-evening-cohort-14/evening-client-side/discussions/2#discussion-48440 while submitting discussion topics.

**Comment by drteresavasquez** [comment, marked as answer]
Take a look at this: https://github.com/nss-evening-cohort-13/student-help/issues/27

**Reply by Mitchell-Crumbley** [reply]
I was unable to get these methods working however I found the VSCode Extension to be very helpful. For anyone coming to this thread:

https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer
